utilities/virt.py

```
@@ -2814,9 +2814,7 @@ def check_virt_handler_pods_for_migration_network(
             f"{Pod.ApiGroup.K8S_V1_CNI_CNCF_IO}/networks", ""
         )
         migration_network_on_pod = pod_network_annotations.split("@")[0] == network_name
-        if migration_network and migration_network_on_pod:
-            verified_pods_list.append(pod)
-        elif not migration_network and not migration_network_on_pod:
+        if migration_network and migration_network_on_pod or not migration_network and not migration_network_on_pod:
```

Ruff (0.15.12)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)